src/app/load-app.js

```diff
@@ -1,16 +1,37 @@
+import { useDataQuery } from '@dhis2/app-runtime'
 import i18n from '@dhis2/d2-i18n'
 import { CircularLoader, CenteredContent, NoticeBox } from '@dhis2/ui'
 import { node } from 'prop-types'
 import React from 'react'
 import { useCustomFormsPrefetch } from '../custom-forms/index.js'
-import { useMetadata } from '../shared/index.js'
+import { useMetadata, useUserInfoStore } from '../shared/index.js'
 import css from './load-app.module.css'
 
+const query = {
```

Comment:
This query is the same as the one in the headerbar (to reuse cache): https://github.com/dhis2/ui/blob/222e21f93a8b0482f94ff6e499ce8b7cdd380824/components/header-bar/src/header-bar.js#L21-L25

Reply:
That's great! I think a comment in the code would be beneficial?